kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/config/AbstractProjectConfig.kt

```diff
@@ -18,19 +19,20 @@ import kotlin.time.Duration
 
 /**
  * Project-wide configuration. Extensions returned by an
- * instance of this class will be applied to all [Spec] and [TestCase]s.
+ * instance of this class will be applied to all [io.kotest.core.spec.Spec] and [io.kotest.core.test.TestCase]s.
```

Comment:
```suggestion
 * instance of this class will be applied to all [Spec] and [io.kotest.core.test.TestCase]s.
```

Since you added an import for Spec.

Reply:
good spot!